README.md

```diff
@@ -1,2 +1,159 @@
-# hello-microverse2
-Hello microverse project
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [Optional] Great for fixing previous reviewer suggestions, I think the logo is broken because you used an unavailable image, please as suggested in the above comment replace the logo with yours if you have one otherwise remove it to have a professional readme 👌